.eslintrc.js

```diff
@@ -163,4 +169,51 @@ module.exports = {
             }
         },
     ],
-};
+});
+
+// Bundled JS
+
+config.overrides.push({
+    files: ['darkreader.js', 'build/debug/chrome/**/*.js'],
+    env: {browser: true},
+    extends: ['plugin:compat/recommended'],
+    parserOptions: {
+        ecmaVersion: 2019,
+        sourceType: 'module'
+    },
+    overrides: [
+
+        // API (modern clients)
+        {
+            files: ['darkreader.js'],
+            rules: {
+
+                // Compatibility check
+                'compat/compat': ['error', [
+                    '>0.5% and supports es5 and supports promises and supports url',
+                    'not Explorer > 0',
+                ].join(', ')]
+            },
+        },
+
+        // Extension (non-mobile browsers based on Firefox or Chromium)
+        {
+            files: ['build/debug/chrome/**/*.js'],
+            rules: {
+
+                // Compatibility check
+                'compat/compat': ['error', [
+                    '> 0.5% and supports es5',
+                    'Firefox ESR',
+                    'not Explorer > 0',
+                    'not Safari > 0',
+                    'not iOS > 0',
+                    'not ChromeAndroid > 0',
+                    'not OperaMini all',
+                ].join(', ')]
```

Comment:
I wanted to make sure.. Are these support rules accurate? English version in the comments. @Gusted

Reply:
Here's the actual list:

```
_darkreader % npx browserslist '>0.5% and supports es5 and supports promises and supports url, not Explorer > 0'
and_chr 98
and_uc 12.12
chrome 97
chrome 96
edge 97
edge 96
firefox 96
firefox 95
ios_saf 15.2-15.3
ios_saf 15.0-15.1
ios_saf 14.5-14.8
ios_saf 14.0-14.4
ios_saf 12.2-12.5
opera 82
safari 15.2-15.3
safari 15.1
safari 14.1
samsung 16.0
_darkreader % npx browserslist '>0.5% and supports es5 Firefox ESR, not Explorer > 0, not Safari > 0, not iOS > 0, not ChromeAndroid > 0, not OperaMini all'
browserslist: Unknown browser query `supports es5 Firefox ESR`. Maybe you are using old Browserslist or made typo in query.
_darkreader % npx browserslist '>0.5% and supports es5, Firefox ESR, not Explorer > 0, not Safari > 0, not iOS > 0, not ChromeAndroid > 0, not OperaMini all'
and_uc 12.12
chrome 97
chrome 96
edge 97
edge 96
firefox 96
firefox 95
firefox 91
opera 82
samsung 16.0

```

Reply:
> The extensions aren't used in mobile, right?

Firefox android has extensions supports.

Reply:
Forgot you'd added this comment, I'll work on this today.

Reply:
Ok here's the new list for the extension code:

```
% npx browserslist '> 0.5% and supports es5, Firefox ESR, last 2 FirefoxAndroid versions, not Explorer > 0, not Safari > 0, not iOS > 0, not ChromeAndroid > 0, notOperaMini all'
and_ff 96
and_uc 12.12
chrome 98
chrome 97
chrome 96
edge 98
edge 97
firefox 97
firefox 96
firefox 91
opera 83
samsung 16.0
```